projects/igniteui-angular/src/lib/directives/tooltip/tooltip-target.directive.ts

```diff
@@ -300,21 +303,40 @@ export class IgxTooltipTargetDirective extends IgxToggleActionDirective implemen
         this._overlayDefaults.closeOnOutsideClick = false;
         this._overlayDefaults.closeOnEscape = true;
 
+        this.target.opening.pipe(takeUntil(this.destroy$)).subscribe(() => {
+            if (this.target.tooltipTarget === this) {
+                document.addEventListener('touchstart', this.onDocumentTouchStart = this.onDocumentTouchStart.bind(this), { passive: true });
+            }
+        });
+
         this.target.closing.pipe(takeUntil(this.destroy$)).subscribe((event) => {
+            if (this.target.tooltipTarget !== this) {
+                return;
+            }
+
+            document.removeEventListener('touchstart', this.onDocumentTouchStart);
             const hidingArgs = { target: this, tooltip: this.target, cancel: false };
             this.tooltipHide.emit(hidingArgs);
 
             if (hidingArgs.cancel) {
                 event.cancel = true;
             }
         });
+
+        this.target.forceClosed.pipe(takeUntil(this.destroy$)).subscribe(() => {
+            document.removeEventListener('touchstart', this.onDocumentTouchStart);
```

Comment:
I'm not sure if it can be expensive to call removeEventListener from all targets, when only one is needed.